lib/MentionsForm/index.js

```diff
@@ -93,14 +93,18 @@ class MentionsForm extends Component {
 
   render() {
     const hasValue = !!this.state.inputValue || !!this.props.value;
-    const hasMention = this.props.disableUntilMention
-      ? this.state.inputValue.search(pattern) !== -1
-      : true;
+    const hasMention =
+      this.state.inputValue.search(pattern) !== -1 ||
+      this.state.inputValue.search(/(\B@\w+)+/gi) !== -1;
+    const disableSubmit = this.props.disableUntilMention
+      ? !hasValue || !hasMention
```

Comment:
I think this can be simplified to
`disableUntilMention ? !hasMention : !hasValue`

Reply:
😅 haha yes!